gravitee-apim-console-webui/src/management/settings/api-logging/api-logging.component.html

```diff
@@ -300,6 +300,27 @@ <h3>Temporal</h3>
       </mat-card-content>
     </mat-card>
 
+    <mat-card class="api-logging__form__card" formGroupName="OpenTelemetry">
```

Comment:
The panel doesn't seem to do anything, nothing is saved when we change the toggle.